geometry_msgs/Pose pose
octomap_msgs/Octomap octomap	
---
int32 value
uint16[] keys 
int32[] values 
geometry_msgs/Point[] frustum